lib/src/StatusEvents/StatusDockShowAppEvent.cpp

```diff
@@ -0,0 +1,45 @@
+#include "DOtherSide/StatusEvents/StatusDockShowAppEvent.h"
+
+#include <QQuickWindow>
+
+
+/*
+    Code here is exactly the same as it was before, logic is not changed. I only 
+    put it in another form, nothing else. To match an improved flow for 
+    installing filters.
+*/
```

Comment:
nit: Probably better to leave this comment in the PR, any change to the file and not the comment could invalidate the statement